src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -174,10 +174,15 @@ body.page-tribe-attendee-registration {
 			/* AR override styles */
 			.tribe-tickets__item {
 				grid-template-columns: 6fr 1fr 2fr 2fr;
+
+				&.tribe-tickets__item--price-sufix {
+					grid-template-columns: 4fr 3fr 2fr 2fr;
```

Comment:
I don't know if grid works out of the box for the autoprefixer in our FE build tool or if we have it working, but something to check. If not, you're using the `-ms` prefixed grid styles in other areas of the code, I'd use it here and anywhere else where we don't have the support for IE11.